pkg/odo/cli/env/env_test.go

```diff
@@ -45,8 +45,20 @@ func TestPrintSupportedParameters(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			got := strings.TrimSpace(printSupportedParameters(tt.supportedParameters))
-			if !reflect.DeepEqual(got, tt.want) {
-				t.Errorf("Got %s, want %s", got, tt.want)
+
```

Comment:
Here's how I would implement it, which I think is more correct:

```suggestion
			gotSet, gotString := getAsSetAndRecomposedString(got)
			wantedSet, wantedString := getAsSetAndRecomposedString(tt.want)
			if !reflect.DeepEqual(wantedSet, gotSet) {
				t.Errorf("Got %s, want %s", gotString, wantedString)
			}
		})
	}
}

func getAsSetAndRecomposedString(output string) (map[string]bool, string) {
	gotStrings := strings.Split(output, "\n")
	gotSet := make(map[string]bool, len(gotStrings))
	for _, s := range gotStrings {
		gotSet[s] = true
	}
	return gotSet, strings.Join(gotStrings, "\n")
}
```

Reply:
I have updated the logic to use DeepEqual and sorting instead.